test/unit/server.js

```diff
@@ -424,8 +413,28 @@ describe('Server', function () {
 
   describe('start', function () {
     describe('without tasks', function () {
-      it('should fail', function () {
-        return assert.isRejected(server.start(), /handler not defined/)
+      beforeEach(function () {
+        sinon.stub(ponosDefaultLogger, 'warn')
+      })
+
+      afterEach(function () {
+        ponosDefaultLogger.warn.restore()
+      })
+
+      it('should warn', function () {
+        server.start().then(function () {
+          sinon.assert.calledTwice(ponosDefaultLogger.warn)
+          sinon.assert.calledWith(
+            ponosDefaultLogger.warn.getCall(0),
```

Comment:
fwiw: I think `ponosDefaultLogger.warn.firstCall` (and `ponosDefaultLogger.warn`) works here as well